src/api/app/views/webui2/shared/_repositories_flag_table.html.haml

```diff
@@ -0,0 +1,26 @@
+.table-responsive
+  %table.table.table-hover
+    %thead.thead-light
+      %tr
+        %th.col-6 Repository
+        %th.col-2.text-center All
+        - @architectures.each do |architecture|
```

Comment:
If you put project / package as local variables why not architecture as well?

Reply:
Sure, oversaw it :+1: